source/solvers/heat_transfer.cc

```diff
@@ -923,7 +990,8 @@ HeatTransfer<dim>::solve_linear_system(const bool initial_step,
   if (simulation_parameters.linear_solver.verbosity !=
       Parameters::Verbosity::quiet)
     {
-      this->pcout << "  -Iterative solver took : " << solver_control.last_step()
+      this->pcout << "  Heat Transfer : " << std::endl
+                  << "  -Iterative solver took : " << solver_control.last_step()
```

Comment:
I don't think it should be at every iterative solver solution that we should announce the physics. Identifying which physics is being solved for should be done by the multiphysics interface (when it calls solve) instead of every solution of a linear system. That would be clearer. You would get something like
Fluid (default, no need to announce)
Heat transfer 
 Newton iteration 0
    - Iterative solver took 
  Newton iteration 1
    - Iterative solver took
Free Surface
Etc.